tests/test_service.py

```diff
@@ -7,6 +8,7 @@
 class TestValidateService(unittest.TestCase):
 
     validate_endpoint = "/validate"
+    healthcheck_endpoint = "/healthcheck"
```

Comment:
Would suggest testing the health check in a separate class `TestHealthcheck` or similar.